js/card.js

```diff
@@ -20,26 +18,10 @@ const getOfferCard = () => {
 
   offerCard.querySelector('.popup__title').textContent = offer.title || '';
   offerCard.querySelector('.popup__text--address').textContent = offer.address || '';
-
-  if (offer.price) {
-    offerCard.querySelector('.popup__text--price').textContent = `${offer.price} ₽/ночь`;
-  } else {
-    offerCard.querySelector('.popup__text--price').textContent = '';
-  }
-
+  (offer.price) ? offerCard.querySelector('.popup__text--price').textContent = `${offer.price} ₽/ночь` : offerCard.querySelector('.popup__text--price').textContent = '';
   offerCard.querySelector('.popup__type').textContent = APARTMENT_TYPES_MAP[offer.type];
-
-  if (offer.rooms && offer.guests) {
-    offerCard.querySelector('.popup__text--capacity').textContent = `${offer.rooms} ${getDeclension(`${offer.rooms}`, ['комната', 'комнаты', 'комнат'])} для ${offer.guests} ${getDeclension(`${offer.guests}`, ['гостя', 'гостей', 'гостей'])}`;
-  } else {
-    offerCard.querySelector('.popup__text--capacity').textContent = '';
-  }
-
-  if (offer.checkin && offer.checkout) {
-    offerCard.querySelector('.popup__text--time').textContent = `Заезд после ${offer.checkin}, выезд до ${offer.checkout}`;
-  } else {
-    offerCard.querySelector('.popup__text--time').textContent = '';
-  }
+  (offer.rooms && offer.guests) ? offerCard.querySelector('.popup__text--capacity').textContent = `${offer.rooms} ${getDeclension(`${offer.rooms}`, ['комната', 'комнаты', 'комнат'])} для ${offer.guests} ${getDeclension(`${offer.guests}`, ['гостя', 'гостей', 'гостей'])}` : offerCard.querySelector('.popup__text--capacity').textContent = '';
```

Comment:
Давай избавимся от дублирования **querySelector** во всех тернанринках. Перенесем тернарник в правую часть выражения
```suggestion
   offerCard.querySelector('.popup__text--capacity').textContent = offer.rooms && offer.guests ? `${offer.rooms} ${getDeclension(`${offer.rooms}`, ['комната', 'комнаты', 'комнат'])} для ${offer.guests} ${getDeclension(`${offer.guests}`, ['гостя', 'гостей', 'гостей'])}` :  '';
```